cmd/main.go

```diff
@@ -60,6 +61,7 @@ func main() {
 			&kustomizecommonv1.Plugin{},
 			&kustomizecommonv2.Plugin{},
 			&declarativev1.Plugin{},
+			&grafana.Plugin{},
 		),
 		cli.WithPlugins(externalPlugins...),
```

Comment:
after the changes, you need to rebase with master and run make install and make generate to fix the testdata.